pkg/yurthub/proxy/util/util.go

```diff
@@ -551,3 +551,37 @@ func ReListWatchReq(rw http.ResponseWriter, req *http.Request) {
 	klog.Infof("this request write error event back finished.")
 	rw.(http.Flusher).Flush()
 }
+
+func IsMultiplexerRequest(req *http.Request, multiplexerResources []schema.GroupVersionResource, nodeName string) bool {
+	ctx := req.Context()
+
+	if req.UserAgent() == util.MultiplexerProxyClientUserAgentPrefix+nodeName {
+		return false
+	}
+
+	info, ok := apirequest.RequestInfoFrom(ctx)
+	if !ok {
+		return false
+	}
+
+	if info.Verb != "list" && info.Verb != "watch" {
+		return false
+	}
```

Comment:
we also need to exclude requests with fieldSelector or labelSelector.

Reply:
Requests with selectors can also be supported, I recommend not excluding them.